src/sass/variables.scss

```diff
@@ -0,0 +1,9 @@
+/**
+  * Vuetify Overrides
+ */
+ @use 'vuetify/settings' with (
+  $body-font-family: (
+    'PT Serif',
+    serif,
+  ),
+ );
```

Comment:
This feels better but will convert the whole app to use this font and I don't think that's out intention. It seems like we only want headers to use this so may need to hit the drawing board and come up with some logical defaults we want to use.

Reply:
I’ll let @itsalaidbacklife check on it but I’m pretty sure it would since it’s a global variable, haven’t tested it though.

Reply:
I'd expect this to set it for the entire app, which would likely be an improvement in that we could hopefully make things more consistent. We should see how it looks, because I do expect this would become a pretty sweeping change. @TommySchmid Could we get some screenshots of the login, home page, and the game view with and without an  overlay?

Reply:
<img width="943" alt="Screen Shot 2023-03-17 at 9 32 20 AM" src="https://user-images.githubusercontent.com/44646343/225919925-7540963f-486d-40e1-b96c-7916eea94f62.png">
<img width="957" alt="Screen Shot 2023-03-17 at 9 32 54 AM" src="https://user-images.githubusercontent.com/44646343/225919933-ec102aae-78cd-446b-b8c0-50adb84b6882.png">
<img width="960" alt="Screen Shot 2023-03-17 at 9 34 50 AM" src="https://user-images.githubusercontent.com/44646343/225919935-db6a2f51-61a3-4a5e-a25b-ca6fdc265a39.png">
<img width="960" alt="Screen Shot 2023-03-17 at 9 34 58 AM" src="https://user-images.githubusercontent.com/44646343/225919940-ad86fa53-031b-43fc-b7ae-20369f72ed92.png">
<img width="958" alt="Screen Shot 2023-03-17 at 9 35 11 AM" src="https://user-images.githubusercontent.com/44646343/225919941-4f4b15c8-cab8-4dfc-8dac-7a707739060d.png">

Reply:
I think this looks good